setup.py

```diff
@@ -130,6 +133,7 @@ def run(self):
         'doc': doc,
         'docker': docker,
         'druid': druid,
+        'gcs': gcs,
```

Comment:
See comment below regarding `gcloud` vs `oath2client`, etc -- for users who don't use GCP, there's no reason to saddle them with extra dependencies. Users who want to use S3 (for anything, not just logs) install `airflow[s3]` and users who want to use GCS should install `airflow[gcs]` or potentially `airflow[gcp]` if the relevant hooks and operators move out of contrib.